packages/sdk/src/types.ts

```diff
@@ -87,6 +87,7 @@ export type AgentMetadata = {
   instructions: string[];
   prompts: string[];
   communicationURL?: string;
+  communicationParams?: object;
```

Comment:
i think we need to add a type, much easier on the subgraph if we already know what properties to look for